civitai-downloader-v2/src/cli/main.py

```
@@ -169,8 +169,552 @@ async def run_search():
+                                version_name = version.get("name", "Unknown")
+                                version_stats = version.get("stats", {})
+                                downloads = version_stats.get("downloadCount", 0) if isinstance(version_stats, dict) else 0
+                                
+                                # Get NSFW status from version or model
+                                nsfw_status = "NSFW" if (version.get("nsfw", False) or result.get("nsfw", False)) else "SFW"
+                                
+                                # Get download URL for this specific version
+                                download_url = f"https://civitai.com/api/download/models/{version_id}"
+                                
+                                # CSV format output using proper CSV writer
+                                output_buffer = io.StringIO()
```

Ruff (0.12.2)

Undefined name `io`

(F821)